nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+
+const reservations = require('./../data/reservations.json');
+
+// /api/reservations
+
+router.get('/', async (request, response) => {
+    try {
+        const allReservations = reservations.map((r) => r);
+        response.send(allReservations);
+    } catch (error) {
+        throw error;
+    }
+});
+
+// /api/reservations/{id}
+
+router.get('/:id', async (request, response) => {
+    try {
+        const id = Number(request.params.id);
+        const allReservations = reservations.find((r) => r.id === id);
+        response.send(allReservations);
```

Comment:
Youre're sending json  so make this ``res.json``.

Here's a little article on what they mean and why use one over the other. [res.json vs res.send](https://www.tutorialspoint.com/difference-between-res-send-and-res-json-in-express-js)